packages/zipkin-instrumentation-request/src/wrapRequest.js

```diff
@@ -0,0 +1,34 @@
+const {Request, Annotation} = require('zipkin');
+
+function wrapRequest(request, {tracer, serviceName = 'unknown', remoteServiceName}) {
+  return request.defaults((options, callback) => {
+    tracer.scoped(() => {
+      tracer.setId(tracer.createChildId());
+      const traceId = tracer.id;
```

Comment:
when you are writing instrumentation, there are two concerns about propagating the traceId:
* ensuring async/error flow are attached to the correct id (manual propagation like you have is fine)
* ensuring downstream library calls are associated with the right id (scoped)

The latter isn't obvious until modification or child the client span is attempted by third-party code. Ex a client span could have a child for TLS setup, for example. If the current traceId wasn't scoped, such child would have an incorrect parent. I don't think we have tests like this at the moment.